src/cache/utils/delete-all-files-from-cache-dir.ts

```diff
@@ -0,0 +1,21 @@
+import { unlink } from 'fs/promises';
+import { CacheDirWithChainId } from '../types';
+import * as gl from 'glob';
+import { promisify } from 'util';
+
+const glob = promisify(gl.glob);
+
+export const deleteAllFilesFromCacheDir = async (
+  cacheDir: CacheDirWithChainId,
+): Promise<void> => {
+  const absoluteFilePaths = await glob(`*`, {
+    cwd: cacheDir,
+    absolute: true,
+  });
+
+  try {
+    await Promise.all(
+      absoluteFilePaths.map(async (filePath) => unlink(filePath)),
+    );
+  } catch (error) {}
```

Comment:
What if we catch a lack of permissions error in this case? Maybe we should output several types of errors to the console?
```js
switch (errorCode) {
  case 'EACCES':
    throw new Error(
      `Permission denied when trying to read the file specified by ${envVarFile}`,
    );
  case 'EMFILE':
    throw new Error(
      `Too many open files in the system when trying to read the file specified by ${envVarFile}`,
    );
  default:
    return
}
```